apps/backend/.env.example

```diff
@@ -41,6 +41,13 @@ AMI_ID=
 REGION=
 SECRET_ACCESS_KEY=
 
+#
+# S3
+#
+S3_BUCKET_NAME=meetlytic-s3
+S3_CONTET_TYPE_AUDIO=audio/mpeg
```

Comment:
Why our backend tells the audio content type? Bot should tell it because bot records audio